### PR TITLE
Project on Authentication using Devise gem and create Post model for checking authentications 

### PR DESCRIPTION
**Pull request by Deepakdanger**
👍 In this Authentication project, we used 'devise' gem
- Here we use devise gem to create a users model.
- we create Post Model with title and body.
💠 The User can only see the post title if he doesn't sign in.
💠 The User can see the post title along with the author if he has already signed in.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<h1> Helllo post index is here</h1>
+
+<ul>
+  <% @posts.each do |post| %>
+    <li>
+    <%= post.title | %>
```
Comment (carlos-ssh) [inline app/views/posts/index.html.erb:6]: ActionView::SyntaxErrorInTemplate in PostsController#index

Comment (carlos-ssh) [inline app/views/posts/index.html.erb]: When I try to run the project appears this error message:
- ActionView::SyntaxErrorInTemplate in PostsController#index

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```
Comment (carlos-ssh) [inline app/models/user.rb:6]: - Add User has*many posts *

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hello team 👋🏽
good job working on this project!

A few changes requested below:
- Add instructions to README in how to run the project locally if it is necessary to download another kind of things... I.E. webpacker
[OPTIONAL]
- A helper method to check if user is signed_in?

**Review by george-shammar — APPROVED** [review]
### **STATUS:APPROVED** :heavy_check_mark: :heavy_check_mark: :+1: 

Good job and thank you for implementing the changes from the first review. :clap: :+1: 
Your project works as expected as well. :+1: 

[OPTIONAL]

- [ ] I strongly advise you to remove all logics from your view. Make use of helper methods instead as was suggested by the previous reviewer.

I wish you goodluck with your future projects. :wave: 

Cheers and happy coding. :smile:

**Pull request merged**